### PR TITLE
drivers/kw2xrf/kw2xrf.c: fix MAX_PKT_LENGTH for RC3

### PR DESCRIPTION
**Pull request by jfischer-phytec-iot**
This patch fixes MAX_PKT_LENGTH on kw2x radio.

### CODE REVIEW

File: drivers/kw2xrf/kw2xrf.c [hunk 1]
```diff
@@ -40,6 +40,8 @@
 #define MKW2XDRF_PA_RANGE_MAX      31       /**< Maximum value of PA Power Control Register */
 #define MKW2XDRF_PA_RANGE_MIN      3        /**< Minimum value of PA Power Control Register */
 
+#define _MAX_MHR_OVERHEAD       (25)
```
Comment (haukepetersen) [inline drivers/kw2xrf/kw2xrf.c]: indention is off

**Comment by haukepetersen** [comment]
ACK when comment is addressed

**Comment by haukepetersen** [comment]
Ups, didn't see that this is the back-port of this PR. So just ignore my comment above...

**Comment by cgundogan** [comment]
Travis is green - GO

**Pull request merged**